ConsoleApp1/Program.cs

```diff
@@ -3,7 +3,44 @@
 	internal class Program
 	{
 		static void Main(string[] args)
-		{
-		}
-	}
+		{
+
+            int[,] array2d = new int[,] 

```

Comment:
The task is following
Create a program that will start with declaration of two constants (X and Y) and will count the sum of all numbers between these constants. If they are equal then sum should be one of them

Example:

X=10
Y=12
Sum=10+11+12=33

X=5
Y=2
Sum=2+3+4+5=14

X=10
Y=10
Sum=10

Extra:

Read values of X and Y from the console. If output is invalid - write to console Invalid input and exit the program.